source/deployment-guide/encryption-options.rst

```diff
@@ -58,6 +58,9 @@ Mattermost supports the following encryption methods for SAML:
 
 * aes128-gcm
 * aes192-gcm
```

Comment:
This cipher was introduced the in same PR as `aes256-gc`

Reply:
Any follow on action you recommend here, @hanzei?